lib/capistrano/scm/git.rb

```diff
@@ -0,0 +1,68 @@
+require "capistrano/plugin"
+require "capistrano/scm"
+
+class Capistrano::SCM::Git < Capistrano::Plugin
+  def set_defaults
+    set_if_empty :git_shallow_clone, false
+    set_if_empty :git_environmental_variables, lambda {
+      {
+        :git_askpass => "/bin/echo",
+        :git_ssh => "#{fetch(:tmp_dir)}/#{fetch(:application)}/git-ssh.sh"
+      }
+    }
```

Comment:
Personally, I find multiline blocks to be easier to read with do...end syntax rather than curly brace syntax. Not sure what the project standard is.